tests/database/test_papis.py

```diff
@@ -31,6 +31,8 @@ def test_database_reload(tmp_library: TemporaryLibrary) -> None:
     assert isinstance(db, papis.database.Database)
 
     ndocs = len(db.get_all_documents())
+
+    assert isinstance(db, papis.database.cache.Database)
```

Comment:
Same here, just remove `db.documents = None`.

Reply:
Still has the problem of `db._save_documents()` not being generic enough.

Reply:
Ah, wait, I didn't notice the file name.. this is meant to only test the `papis` database backend! Then yeah, this is perfectly fine.. sorry for the noise.